### PR TITLE
Fix various typos (Fixes #27396)

### LINKED ISSUE #27396: [BUG] Typos
There are a variety of typos in the repo.  Fixed by https://github.com/microsoft/playwright/pull/27391

### System info
- Playwright Version: All
- Operating System: All
- Browser: All
- Other info:

### Source code
n/a

### PR DESCRIPTION
**Pull request by calvinballing**
I ran cspell locally, and fixed various typos.  The first commit affects:
- documentation (code comments, test descriptions, etc.)
- variable names (changes made consistently across all instances of the variable)

The second commit fixes an error in some sample code caused by a typo

Fixes https://github.com/microsoft/playwright/issues/27396

### CODE REVIEW

**Comment by calvinballing** [comment]
@microsoft-github-policy-service agree

File: tests/assets/reading-list/vue_3.1.5.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-var Vue = (function (exports) {
-  'use strict';
-
+var Vue = (function (exports) {
```
Comment (pavelfeldman) [inline tests/assets/reading-list/vue_3.1.5.js:1]: Let's revert this one.

**Comment by pavelfeldman** [comment]
This looks great, thanks!

**Comment by calvinballing** [comment]
> This looks great, thanks!

@pavelfeldman I addressed the callout above, and we're now back to all green!

**Comment by pavelfeldman** [comment]
Thanks! So many files. I made another pass on those. Please revert the following paths:

```yml
# Patches are derived resources
browser_patches/

# We can't touch third party
packages/playwright-core/src/third_party

# This file is generated
packages/playwright-core/types/protocol.d.ts

# These are third party resources
tests/assets/reading-list/
```

**Comment by calvinballing** [comment]
> Thanks! So many files. I made another pass on those. Please revert the following paths:

Thanks for the extra look-through!  I've made the changes mentioned above

**Review by pavelfeldman — APPROVED** [review]

**Comment by pavelfeldman** [comment]
I'll follow up w/ generated types.

**Pull request merged**